### ISSUE TITLE
[Tabs] Missing key ARIA role="tablist"

### ISSUE THREAD

**Issue by gerardkcohen** [zurb/foundation-sites#9060]
Tab container is missing key parent `role="tablist"` which may produce compatibility issues with various assistive technology/ browser combinations.

As specified in the official WAI-ARIA Authoring Practices for [Tab Panel](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel):

> A tablist is the container role for a set of elements with the role attribute set to tab.

This can be seen on current [Tabs documentation](http://foundation.zurb.com/sites/docs/tabs.html) by inspecting any Tab `<ul>` container.

**Comment by abdullahsalem** [comment]
@kball Should this be closed?

**Comment by kball** [comment]
yes. Fix will be in 6.3
